conans/client/generators.py

```diff
@@ -84,6 +96,9 @@ def _aux_cmake_test_setup(self):
     SET(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} ${CONAN_C_FLAGS}")
     SET(CMAKE_SHARED_LINK_FLAGS "${CMAKE_SHARED_LINK_FLAGS} ${CONAN_SHARED_LINK_FLAGS}")
     SET(CMAKE_EXE_LINKER_FLAGS "${CMAKE_EXE_LINKER_FLAGS} ${CONAN_EXE_LINKER_FLAGS}")
+    SET(CMAKE_MODULE_PATH ${CONAN_CMAKE_MODULE_PATH} ${CMAKE_MODULE_PATH})
```

Comment:
I would say not to put CMAKE_MODULE_PATH here, inside the "flags" method, but it deserves its own line in CONAN_BASIC_SETUP()